doc/nrf/known_issues.rst

```diff
@@ -355,6 +355,18 @@ LTE Sensor Gateway fails to run
 
     west build --board nrf9160dk_nrf9160_ns@1.1.0
 
+.. rst-class:: v2-2-0 v2-1-2 v2-1-1 v2-1-0
+
+CIA-738: Partition external flash for FMFU in samples/applications
```

Comment:
```suggestion
CIA-738: Partition external flash for FMFU in samples and applications
```
By the way, what is the issue here? The initial line always states the issue.

Reply:
I reworded it.